operator/pkg/ciliumendpointbatch/manager.go

```diff
@@ -0,0 +1,337 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ciliumendpointbatch
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/cilium/cilium/pkg/annotation"
+	cilium_v2 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2alpha1"
+	"github.com/cilium/cilium/pkg/lock"
+)
+
+var (
+	// sequentialLetters contains lower case alphabets without vowels and few numbers.
+	// skipped vowels and numbers [0, 1] to avoid generating controversial names.
+	sequentialLetters = []rune("bcdfghjklmnpqrstvwxyz2456789")
+)
+
+// ceBatch holds local copy of CiliumEndpointBatch.
+type ceBatch struct {
```

Comment:
nit: name is a bit odd...maybe just batch?

Reply:
I don't see any strange here, prefix `ce` implies batch for `CiliumEndpoint`

Reply:
> I don't see any strange here, prefix `ce` implies batch for `CiliumEndpoint`

It is strange because it is half acronym and half full name...

Reply:
this isn't user facing resource. This implies internal structure name.

Reply:
Still strange, could be better named for code readability.